aten/src/ATen/native/quantized/cpu/qnnpack_utils.h

```diff
@@ -29,21 +32,22 @@ struct PackedLinearWeightsQnnp : public LinearPackedParamsBase {
       at::Tensor orig_weight,
       at::Tensor bias,
       c10::optional<double> input_scale,
-      double w_scale,
-      int64_t w_zp)
+      at::Tensor w_scales,
+      at::Tensor w_zps)
       : w(std::move(w)),
         orig_weight(std::move(orig_weight)),
         bias_(std::move(bias)),
         input_scale(std::move(input_scale)),
-        w_scale(w_scale),
-        w_zp(w_zp) {}
+        w_scales(w_scales),
+        w_zero_points(w_zps) {}
 
   std::unique_ptr<qnnpack::PackBMatrix> w;
   at::Tensor orig_weight;
   at::Tensor bias_;
   c10::optional<double> input_scale;
-  double w_scale;
-  int64_t w_zp;
+  at::Tensor w_scales;
+  at::Tensor w_zero_points;
```

Comment:
Why take these as at::Tensor rather than std::vector?

Reply:
I think this can be made std::vector. Let me check.